### ISSUE TITLE
Navigation config does not reflect navigation menu

### ISSUE THREAD

**Issue by pinclau** [dotkernel/dot-admin#299]
Changing something in navigation.global.php does not reflect in the admin left menu from admin, it seems that is not used at all

**Comment by alexmerlin** [comment]
It's because when the new theme was implemented, the navigation was hardcoded instead of reusing the existing implementation.
https://github.com/dotkernel/admin/blob/5.0/src/App/templates/partial/left-menu.html.twig

**Comment by pinclau** [comment]
Yes, i saw that but i was tinking about fix it or delete it, why require dot-navigation and keep those config file if they are not used.

**Comment by alexmerlin** [comment]
Good question... but, I'm not the author of the new theme implementation.

I agree that we should either:

- reimplement dot-navigation (recommended)
- remove both dot-navigation and its config file, to avoid further confusions

**Comment by arhimede** [comment]
Reimplement dot-navigation seem a good direction

@bidi47 maybe ?

**Comment by arhimede** [comment]
or maybe @MarioRadu  ?
